### PR TITLE
Reauth local : added new password reset functionality

### PR DESCRIPTION
**Pull request by abhikapoor**

### CODE REVIEW

File: routes/signup.js [hunk 1]
```diff
@@ -144,6 +149,20 @@ module.exports = function(absession) {
 
         console.log('[RE-Signup] UPDATE username setup: ', req.body)
 
+        try {
+            verify_res = await this.absession.call('sys.account.check_username', [{
```
Comment (markope) [inline routes/signup.js:153]: there is no check_username function and it is not neede. I think we can remove this section.

Comment (abhikapoor) [reply routes/signup.js:153]: my old changes i forgot to remove

Comment (markope) [reply routes/signup.js:153]: thought so

Comment (abhikapoor) [reply routes/signup.js]: pushed

**Pull request merged**